docs/advanced/api.rst

```diff
@@ -7,6 +7,19 @@ Flocker REST API
 We are currently in the process of developing an external HTTP-based REST API for Flocker.
 While this API is not yet directly accessible in a standard Flocker setup, the documentation below will give a sense of what will eventually be available.
 
+
+Installation
+============
+
+To allow remote access to the control agent REST API, and for agent connections,
```

Comment:
Maybe make more explicit that this refers to platforms using `firewalld` like CentOS or Fedora.